### ISSUE TITLE
Release Skywork UniPic Curated Training Dataset on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [SkyworkAI/UniPic#13]
Hi @SkyworkAI 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2508.03320.
The paper page lets people discuss about your paper and lets them find artifacts about it (your dataset for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

It's great to see that the `Skywork/Skywork-UniPic-1.5B` model is already hosted on the Hugging Face Hub! This is fantastic for discoverability.

I also noticed in your abstract that you mention "meticulously curated, 100 million-scale datasets augmented with task-specific reward models" for training UniPic. Would you be interested in hosting these datasets on https://huggingface.co/datasets?
Hosting on Hugging Face will give you more visibility/enable better discoverability, and will also allow people to do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```

If you're down, leaving a guide here: https://huggingface.co/docs/datasets/loading.
We also support Webdataset, useful for image/video datasets: https://huggingface.co/docs/datasets/en/loading#webdataset.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

After uploaded, we can also link the datasets to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

Let me know if you're interested/need any guidance.

Kind regards,

Niels

**Comment by Orlando-CS** [comment]
Hi Niels,

Thanks a lot for reaching out and for featuring our paper on Hugging Face’s daily papers — we really appreciate the visibility and support from the open-source community.
Regarding the dataset: while we would love to share our training data to further promote openness and reproducibility, unfortunately, the curated 100M-scale datasets used for UniPic are not approved for public release due to internal company data governance policies.

**Comment by NielsRogge** [comment]
Makes sense, thanks for your reply. Really nice to see the model released on HF though :)
